uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/main/java/org/uberfire/java/nio/fs/jgit/JGitBasicAttributeView.java

```diff
@@ -71,26 +71,23 @@ private BasicFileAttributes buildAttrs(final JGitFileSystem fs,
 
         final Ref ref = fs.getGit().getRef(branchName);
 
-        final List<Date> records = new ArrayList<>();
+        final List<RevCommit> records = new ArrayList<>();
+        final Long[] result = new Long[2];
```

Comment:
Lets create a new class to use names like "first commit date" and "last commit date" (or something like that) instead of thinking in a Long[].

Reply:
@RishiRajAnand As this comes from your initial PR, could you please take care of replacing the Long[]? Thanks!

Reply:
sure will do